pkg/util/sqlkiller/sqlkiller_test.go

```
@@ -0,0 +1,110 @@
+	}
+
+	t.Run("reset after successful kill signal CAS", func(t *testing.T) {
+		killer := &SQLKiller{}
+		core, logs := observer.New(zap.WarnLevel)
+		restoreLogger := log.ReplaceGlobals(zap.New(core), &log.ZapProperties{
+			Core:  core,
+			Level: zap.NewAtomicLevelAt(zap.WarnLevel),
+		})
+		defer restoreLogger()
+
+		const beforeLogFailpoint = "github.com/pingcap/tidb/pkg/util/sqlkiller/" +
```

golangci-lint (2.12.2)

[warning] line-length-limit: line is 82 characters, out of limit 80

(revive)